DOCKER-network/stop.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# shellcheck disable=SC2046
+docker container kill $(docker ps -q)
+docker system prune -f
```

Comment:
Can we just remove this specific network instead of using system prune

Reply:
Yea I can look into that, I was assuming that developers would run one network on their machine at a time.